src/main/resources/palette_pane.fxml

```diff
@@ -13,56 +13,20 @@
   <children>
     <VBox alignment="TOP_CENTER" prefHeight="200.0" prefWidth="100.0" spacing="8.0" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
       <children>
-        <ToggleButton mnemonicParsing="false" onAction="#selectPencil">
-          <graphic>
-            <ImageView fitHeight="32.0" fitWidth="32.0" pickOnBounds="true" preserveRatio="true">
-              <image>
-                <Image url="@images/light/pixels.png" />
-              </image>
-            </ImageView>
-          </graphic>
-          <tooltip>
-            <Tooltip text="Pencil Tool" />
-          </tooltip>
-          <toggleGroup>
-            <ToggleGroup fx:id="palette" />
-          </toggleGroup>
-        </ToggleButton>
-        <ToggleButton mnemonicParsing="false" onAction="#selectEraser" toggleGroup="$palette">
-          <graphic>
```

Comment:
format the file

Reply:
It’s a little bit scary to format FXML files if others have made changes to them. But I’ll format them.